drivers/doc.txt

```diff
@@ -43,3 +43,10 @@
  * @ingroup     drivers
  * @brief       Drivers for storage devices
  */
+
+/**
+ * @defgroup    drivers_can CAN Drivers
+ * @ingroup     drivers
+ * @ingroup     can
```

Comment:
LoRa is a seperate stack, too. This doesn't imply that it needs a whole new group. If `can` devices don't fit into `drivers_netdev` for some reason (because CAN isn't networking (?)) how about renaming it to something like "communication devices"?